packages/relay-compiler/util/md5.js

```diff
@@ -0,0 +1,11 @@
+//@flow
```

Comment:
This file probably needs a FB header like all other files.

Reply:
Done